srcpkgs/girara/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'girara'
 pkgname=girara
-version=0.3.1
+version=0.3.2
 revision=1
 build_style=meson
-hostmakedepends="pkg-config intltool"
+configure_args="-Dnotify=auto -Dtests=disabled"
+hostmakedepends="pkg-config intltool doxygen"
 makedepends="gtk+3-devel libnotify-devel glib-devel json-c-devel"
 short_desc="A library implementing a user interface that focuses on minimalism"
 maintainer="lemmi <lemmi@nerd2nerd.org>"
 license="Zlib"
 homepage="http://pwmt.org/projects/girara/"
 distfiles="http://pwmt.org/projects/${pkgname}/download/${pkgname}-${version}.tar.xz"
-checksum=3c409bac00b6628aa80c601a64e11c3ff07a3c0721ac80436eceb282cb55bcb5
+checksum=1700353a101f3c520f9b22e79d71ea5b268a9ec324796cf9e64775d96bb086cd
 
 if [ "$CROSS_BUILD" ]; then
 	hostmakedepends+=" glib-devel"
```

Comment:
i'd just add glib-devel to hostmakedepends regardless.

libglib-devel is meant for makedepends= and glib-devel for hostmakedepends, regardless of native or cross

Reply:
ah, good to know. on it.